constrained_path_generator/PlanConstrainedPathQuery query
---
constrained_path_generator/PlanConstrainedPathResult result
